src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -164,13 +166,14 @@ BaseCreateCourseForm.propTypes = {
     name: PropTypes.string.isRequired,
     key: PropTypes.string.isRequired,
   })).isRequired,
-  pristine: PropTypes.bool.isRequired,
-  submitting: PropTypes.bool.isRequired,
+  pristine: PropTypes.bool,
+  submitting: PropTypes.bool,
   isCreating: PropTypes.bool,
 };
 
 const CreateCourseForm = reduxForm({
   form: 'create-course-form',
 })(BaseCreateCourseForm);
-export { BaseCreateCourseForm, CreateCourseForm };
+export default CreateCourseForm;
```

Comment:
no need to change just an fyi, you can just do export default CreateCourseForm = reduxForm...

Reply:
Ok, I thought it would be more clear to have both exports below.  I will adjust.

Reply:
Need to rebase anyway